atomium/templates/user_picture/user_picture.component.inc

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * @file
+ * Contains component file.
+ */
+
+/**
+ * Implements hook_atomium_theme_hook().
+ */
+function atomium_atomium_theme_user_picture() {
+  return array(
+    'user_picture' => array(
+      'template' => 'user-picture',
+      'variables' => array(
+        'account' => NULL,
+        'user_picture' => NULL,
+      ),
+    ),
+  );
+}
+
+/**
+ * Implements hook_preprocess_hook().
+ */
+function atomium_preprocess_user_picture(&$variables, $hook) {
+  if (variable_get('user_pictures', 0)) {
+    $account = $variables['account'];
+
+    if (!empty($account->picture)) {
+      // @TODO: Ideally this function would only be passed file objects, but
```

Comment:
is this TODO still useful? if yes its purpose is not pretty clear...

Reply:
It the comment from the template_preprocess_comment() function. 

I don't think it hurts to leave it there as we've copy paste the code. 

Now, I will ask the team what they think and give you feedback asap. 

Thanks.